store/interfaces.json

```diff
@@ -47,6 +48,7 @@
       "minicart",
       "login",
       "search-bar",
+      "locale-switcher",
```

Comment:
You should create an independent block for `locale-switcher` as well.

So it will work with the `header-layout`